README.md

```diff
@@ -4,6 +4,113 @@
 
 This library provides instrumentation for CDI modules in Java EE applications.
 
+## Usage
+
+For a concrete and comprehensive set of examples, refer to the `opentracing-cdi-example`
+module.
+
+### Injecting a tracer
+
+In your managed beans, you can get an instance of the appropriate tracer via:
+
+```java
+@Inject
+Tracer tracer;
+```
+
+If you have your own `Tracer` producer, you'll need to mark your own as an
+`@Alternative`
```

Comment:
> Have a qualified producer for Tracer, requiring developers to use this qualifier every time they want to use our producer.

Do you mean they would have to use that annotation when injecting the tracer? 

> I believe that our producer will be the most commonly used,

I would personally write my own producer which would return a configured tracer